usr/include/pkcs32.h

```diff
@@ -169,10 +169,10 @@ typedef struct CK_ATTRIBUTE_32 {
     CK_ULONG_32 ulValueLen;     /* in bytes */
 } CK_ATTRIBUTE_32;
 
-
-
+/* AIX compiles with -qalign=natural */
+#if !defined(_AIX)
 #pragma pack()
```

Comment:
I wonder if this is required at all... What effect does it have if it appears at the end of the header file. It won't affect the structures defined in that header file. It might though have an effect on structures that are defined after the inclusion of this header file.... I wonder if thats expected....

According to the GCC doc, `#pragma pack()` (without anything inside the brackets) sets the alignment to the one that was in effect when compilation started. Since there is no pragma pack anywhere that sets a different alignment, it has essentially no effect, imho.

So I guess we can completely remove that.